src/Symfony/Component/Mailer/composer.json

```diff
@@ -17,13 +17,13 @@
     ],
     "require": {
         "php": "^7.1.3",
+        "egulias/email-validator": "^2.0",
```

Comment:
This would add a dependency on a non-organizational repository outside of Symfony, which can cause all sorts of issues:
 - Bus Factor
 - Possibly slow patching of (security) bugs
 - less control over releases and how fast they can be done

I don't oppose using this package as a required dependency (though I agree with nicolas' alternative), just want to make sure the above issues (and possibly others) won't cause problems.

Reply:
Swiftmailer has 110M downloads and has been using this library for years. Drupal and Laravel are also using this library.